main/checker-node.js

```diff
@@ -76,24 +76,24 @@ function maybeReportErrorToSentry (err, scopeFn) {
  * @param {Context} ctx
  */
 async function start (ctx) {
-  log.info('Starting Core...')
+  log.info('Starting Checker...')
 
   const childProcess = fork(
-    corePath,
-    ['--json', '--recreateStationIdOnError'],
+    checkerNodePath,
+    ['--json', '--recreateCheckerIdOnError'],
     {
       env: {
         ...process.env,
         FIL_WALLET_ADDRESS: await wallet.getAddress(),
-        PASSPHRASE: await wallet.signMessage('station core passhprase'),
+        PASSPHRASE: await wallet.signMessage('checker node passhprase'),
```

Comment:
This is a breaking change. The Checker Node won't be able to decrypt the Station ID file encoded using the old passphrase derived from "station core passhphrase". Since we are using `--recreateCheckerIdOnError`, the Checker Node will create a new checker id.

Is that your intent?

While the upgrade is rolled to the network, we will see twice as many Checker IDs in our telemetry.

Are we okay with that?

My suggestion is to keep the old passphrase for backward compatibility.

```suggestion
        // Note: we must keep the passphrase to avoid rotation of Checker IDs
        PASSPHRASE: await wallet.signMessage('station core passhprase'),
```

OTOH, if we decide to change the passphrase, then let's choose a new value that we won't need to change in the future.

```suggestion
        PASSPHRASE: await wallet.signMessage('passhprase seed'),
```

Reply:
My thinking was that a one time rotation of all ids shouldn't be a problem, but I agree that simply keeping the old pass phrase makes reporting simpler, while changing the passphrase doesn't add any value. I'll revert that part

Reply:
a48603a